importer/keycloak/export_kc.go

```diff
@@ -0,0 +1,126 @@
+package keycloak
+
+import (
+	"database/sql"
+	"encoding/json"
+	"fmt"
+	"github.com/Bnei-Baruch/mdb/common"
+	"github.com/Bnei-Baruch/mdb/models"
+	"github.com/Bnei-Baruch/mdb/utils"
+	log "github.com/Sirupsen/logrus"
+	"github.com/pkg/errors"
+	"github.com/spf13/viper"
+	"github.com/volatiletech/sqlboiler/boil"
+	"github.com/volatiletech/sqlboiler/queries/qm"
+	"gopkg.in/volatiletech/null.v6"
+	"io/ioutil"
+	"net/http"
+	"path"
+	"strings"
+)
+
+type ExportKC struct {
+	mdb  *sql.DB
+	resp []*RespItem
+}
+
+type KCData struct {
+	AccountID string `json:"id"`
+	FName     string `json:"firstName"`
+	LName     string `json:"lastName"`
+}
+
+type RespItem struct {
+	email    string
+	wasAdded bool
+	error    error
+}
+
+func (e *ExportKC) Run() {
+	e.mdb = e.openDB()
+	defer e.mdb.Close()
+
+	users, err := models.Users(e.mdb, qm.Where("account_id IS NULL")).All()
+	utils.Must(err)
+	e.resp = make([]*RespItem, len(users))
+	withErr := make([]*models.User, 0)
+	for i, u := range users {
+		e.resp[i] = &RespItem{
+			email:    u.Email,
+			wasAdded: true,
+			error:    nil,
+		}
+		if err := e.updateUser(u); err != nil {
+			e.resp[i].wasAdded = false
+			e.resp[i].error = err
+			withErr = append(withErr, u)
+			log.Debugf("Error for email: %s Error: %v ", u.Email, err)
+		}
+	}
+	e.useEmailAsAccountId(withErr)
```

Comment:
what you say? have few emails that no on keycloak

Reply:
I'm fine with that. How many such users we have like this ?

Reply:
5-6